<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectWindow</class>
 <widget class="QWidget" name="ProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,3">
   <property name="spacing">
    <number>3</number>
   </property>
   <item>
    <widget class="QListWidget" name="fileList"/>
   </item>
   <item>
    <widget class="Line" name="projectVerticalLine">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="metadataVerticalLayout" stretch="1,0,3">
     <item>
      <layout class="QHBoxLayout" name="constantMetadataHorizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="constantMetadataLeftVerticalLayout">
         <item>
          <widget class="QLabel" name="projectNameLabel">
           <property name="text">
            <string>Project Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="projectNameEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="dateLabel">
           <property name="text">
            <string>Experiment Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="constantMetadataRightVerticalLayout">
         <item>
          <widget class="QLabel" name="noteLabel">
           <property name="text">
            <string>Note</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="noteEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="metadataMainHorizontalLine">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="mutableMetadataVerticalLayout" stretch="1,0,999">
       <item>
        <widget class="QLabel" name="mutableMetadataHeader">
         <property name="text">
          <string>Additional data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="headerHorizontalLine">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="mutableMetadataMainVerticalLayout" stretch="1,100">
         <item>
          <layout class="QGridLayout" name="keyValueGridLayout" columnstretch="1,999">
           <item row="0" column="0">
            <widget class="QPushButton" name="addMetadataButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QFrame" name="mutableMetadataFiller">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="keyValueList"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
